experimental/packages/opentelemetry-instrumentation-http/test/functionals/utils.test.ts

```diff
@@ -320,6 +321,31 @@ describe('Utility', () => {
       assert.deepEqual(attributes[SemanticAttributes.HTTP_ROUTE], undefined);
     });
   });
+
+  describe('getIncomingRequestMetricAttributesOnResponse()', () => {
+    it('should correctly add http_route if span has it', () => {
+      const spanAttributes: Attributes = {
+        [SemanticAttributes.HTTP_ROUTE]: '/user/:id',
+      };
+      const metricAttributes =
+        utils.getIncomingRequestMetricAttributesOnResponse(spanAttributes);
+
+      assert.deepStrictEqual(
+        metricAttributes[SemanticAttributes.HTTP_ROUTE],
+        '/user/:id'
+      );
+    });
+
+    it('should skip http_route if span has not it', () => {
```

Comment:
Just a nit - feel free to skip it. :slightly_smiling_face: 
```suggestion
    it('should skip http_route if span does not have it', () => {
```